### PR TITLE
Repair arangorestore.

### PR DESCRIPTION
**Pull request by neunhoef**
Write requests from coordinator to leaders now have a longer timeout
to allow for stopped writes by a follower trying to get in sync.

Also improve error logging in the `restore` REST handler.

### CODE REVIEW

File: arangod/RestHandler/RestReplicationHandler.cpp [hunk 1]
```diff
@@ -1398,16 +1398,35 @@ Result RestReplicationHandler::processRestoreDataBatch(
...
       return opRes.result;
     }
+    if (duration > 30) {
+      LOG_TOPIC(INFO, Logger::PERFORMANCE) << "Restored/deleted "
+        << oldBuilder.slice().length() << " documents in time: " << duration
+        << " seconds.";
+    }
   } catch (arangodb::basics::Exception const& ex) {
+    LOG_TOPIC(WARN, Logger::CLUSTER)
+      << "Could not delete documents for restore exception (1): "
```
Comment (jsteemann) [inline arangod/RestHandler/RestReplicationHandler.cpp]: Can we remove the extra (1) (2) and (3) suffixes here?
These are user-facing messages, and the suffixes provide no value to the user. To us, they will also not provide any value, except that we know the type of the exception, which is probably not super-interesting.

File: arangod/RestHandler/RestReplicationHandler.cpp [hunk 2]
```diff
@@ -1478,15 +1497,34 @@ Result RestReplicationHandler::processRestoreDataBatch(
...
       return opRes.result;
     }
+    if (duration > 30) {
+      LOG_TOPIC(INFO, Logger::PERFORMANCE) << "Restored/inserted "
+        << requestSlice.length() << " documents in time: " << duration
+        << " seconds.";
+    }
   } catch (arangodb::basics::Exception const& ex) {
+    LOG_TOPIC(WARN, Logger::CLUSTER)
+      << "Could not insert documents for restore exception (1): "
```
Comment (jsteemann) [inline arangod/RestHandler/RestReplicationHandler.cpp]: same here

File: arangod/RestHandler/RestReplicationHandler.cpp [hunk 3]
```diff
@@ -1527,15 +1565,34 @@ Result RestReplicationHandler::processRestoreDataBatch(
...
       return opRes.result;
     }
+    if (duration > 30) {
+      LOG_TOPIC(INFO, Logger::PERFORMANCE) << "Restored/inserted "
+        << replBuilder.slice().length() << " documents in time: " << duration
+        << " seconds.";
+    }
   } catch (arangodb::basics::Exception const& ex) {
+    LOG_TOPIC(WARN, Logger::CLUSTER)
+      << "Could not replace documents for restore exception (1): "
```
Comment (jsteemann) [inline arangod/RestHandler/RestReplicationHandler.cpp]: and once again

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-matrix-pr-linux/2734/

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/1736/

**Pull request merged**